src/chapters/classes-part2/equals-shortcut.rst

```diff
@@ -0,0 +1,72 @@
+Visual Studio Generator Shortcut
+================================
+
+Instead of cutting, pasting, and refactoring old code to ensure that you create a well-structured ``GetHashCode()`` method whenever you define your own ``Equals()`` method, you can use Visual Studio’s code generation tool!
+Just right-click in your class file on the class name and follow these prompts.
+
+Let’s use a Course class to demonstrate:
+
+.. sourcecode:: csharp
+   :linenos:
+
+   public class Course
+   {
+      public string Topic { get; set; }
+      public Teacher Instructor { get; set; }
+      public List<Student> enrolledStudents { get; set; }
+   }
+
+#. In Visual Studio, right-click on the class name and select *Quick Fix* for Mac users or *Quick Actions and Refactorings* for Windows users from the menu.
```

Comment:
i don't need quick-fix fyi. we can chat about this in person

Reply:
I added another method